### PR TITLE
Prompt for python path

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
Fixes https://github.com/Microsoft/vscode-azurefunctions/issues/594
screen shot 2018-09-13 at 10 57 40 am
screen shot 2018-09-13 at 10 57 48 am

I decided not to validate _after_ the user enters a custom path. I figure we let them use what they want to use and they will get error message down the line if it doesn't work.

### CODE REVIEW

**Comment by PrashanthCorp** [comment]
If you're trading an error now for an error later, do you have examples of what the errors could be? Just wondering if they're descriptive enough about requiring the right python version.

**Comment by EricJizbaMSFT** [comment]
@PrashanthCorp I decided it was more work to get the error messages to show you than to just implement more validation logic

File: src/commands/createNewProject/PythonProjectCreator.ts [hunk 1]
```diff
@@ -191,3 +195,24 @@ export class PythonProjectCreator extends ScriptProjectCreatorBase {
...
 }
+
+/**
+ * Returns undefined if valid or an error message if not
+ */
+async function validatePythonAlias(pyAlias: PythonAlias): Promise<string | undefined> {
+    try {
+        const result: cpUtils.ICommandResult = await cpUtils.tryExecuteCommand(undefined /*don't display output*/, undefined /*default to cwd*/, `${pyAlias} --version`);
+        if (result.code !== 0) {
+            return localize('failValidate', 'Failed to validate version:{1}{2}', pyAlias, os.EOL, result.cmdOutputIncludingStderr);
```
Comment (StephenWeatherford) [inline src/commands/createNewProject/PythonProjectCreator.ts]: You missing an arg?

Comment (EricJizbaMSFT) [reply src/commands/createNewProject/PythonProjectCreator.ts]: Cleaned up the code. Screenshot above is how I intended the message to look

**Pull request merged**